src/components/Header.js

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
-import './Header.css';
+import './../css/Header.css';
 
 const Header = () => {
   return (
     <div className="ui header-size menu">
-      <img src={require('./logo.png')} alt="logo" />
+      <img src={require('./../assets/images/logo.png')} alt="logo" />
```

Comment:
Instead of using `require('./../assets/images/logo.png')` you could change it to the following way:

```js
import React from 'react';
import './../css/Header.css';
import logo from './../assets/images/logo.png';

const Header = () => {
    return (
        <div className="ui header-size menu">
            <img src={logo} alt="logo" />
```